editor/plugins/script_text_editor.cpp

```diff
@@ -1628,6 +1628,13 @@ void ScriptTextEditor::_color_changed(const Color &p_color) {
 	code_editor->get_text_editor()->update();
 }
 
+void ScriptTextEditor::_prepare_edit_menu() {
+	const CodeEdit *tx = code_editor->get_text_editor();
+	PopupMenu *popup = edit_menu->get_popup();
+	popup->set_item_disabled(popup->get_item_index(EDIT_UNDO), !tx->has_undo());
+	popup->set_item_disabled(popup->get_item_index(EDIT_REDO), !tx->has_redo());
+}
```

Comment:
Any reason behind having this on `about_to_popup`? should be fine with the code in `_make_context_menu`. Or we could remove the code in `_make_context_menu` and rely on this, no need to do the same thing twice.